charts/stable/double-take/README.md

```diff
@@ -0,0 +1,118 @@
+# double-take
+
+![Version: 1.0.0](https://img.shields.io/badge/Version-1.0.0-informational?style=flat-square) ![AppVersion: 1.0.0](https://img.shields.io/badge/AppVersion-1.0.0-informational?style=flat-square)
```

Comment:
Appversion appears to really be 1.16.0

Reply:
Dont forget to regen readme :D

Reply:
it's actually 1.6.0 from what i can tell; updated!